autoPyTorch/ensemble/abstract_ensemble.py

```diff
@@ -76,3 +80,12 @@ def get_validation_performance(self) -> float:
         Returns:
             Score
         """
+
+    def update_identifiers(
+        self,
+        replace_identifiers_mapping: Dict[Tuple[int, int, float], Tuple[int, int, float]]
+    ) -> None:
+        identifiers = copy(self.identifiers_)
```

Comment:
it is super minor, but we could do:
```suggestion
        identifiers = self.identifiers_.copy()
```